schema_utils.py

```diff
@@ -338,6 +338,18 @@ def is_nonempty(obj):
         """
         return bool(obj)
 
+    @staticmethod
+    def is_valid_url(obj):
```

Comment:
Can we change it to something like `is_mail_to_absent` since this is one part of a valid url, there is one other validator `is_nonempty` which is used in url validation.